Table Printer

```diff
@@ -0,0 +1,37 @@
+
+""" Write a function named printTable() that takes a list of lists of strings and displays it in a well-organized table
+with each column right-justified. Assume that all the inner lists will contain the same number of strings. For
+example, the value could look like this:
+
+tableData = [['apples', 'oranges', 'cherries', 'banana'],
+['Alice', 'Bob', 'Carol', 'David'],
+['dogs', 'cats', 'moose', 'goose']]
+
+Your printTable() function would print the following:
+apples Alice dogs
+oranges Bob cats
+cherries Carol moose
+banana David goose """
+
+
+tableData = [['apples', 'oranges', 'cherries', 'banana'], ['Alice', 'Bob', 'Carol', 'David'],
+             ['dogs', 'cats', 'moose', 'goose']]
+
+
+def printtable(my_list):
+    max_len = 0
+    for item in my_list:
+        for item_2 in item:
```

Comment:
For readability, aim to use better names for variables. I.e. instead of item and item2, array and item